### DISCUSSION TITLE
moving to scss?

### DISCUSSION THREAD

**Discussion by ToBinio** [rust-lang/crates.io#6462] [General]
In my opinion, the most important feature of scss is already enabled through `postcss-nested`. But sadly I did not get my ide (Webstorm | Clion) to understand this. changing the file ending to scss solved it.

Maybe I am missing something but I do think moving to scss should be a more or less simple fix.

**Comment by Turbo87** [comment]
we are trying to move to vanilla(ish) CSS instead of using SASS/SCSS. `postcss-nested` is currently still used until CSS has native nesting syntax shipped.

**Reply by ToBinio** [reply]
sorry i forgot the check if there where ever something similar and I just noticed that you moved away from scss about 2 years ago....

just for my curiosity... is there any specific reason why crates.io uses mostly native stuff (besides ember.js)?

**Reply by Turbo87** [reply]
> is there any specific reason why crates.io uses mostly native stuff (besides ember.js)?

basically just for simplicity. I've seen a lot of codebases with unnecessary accidental complexity introduced by using things like SASS. it also increases compile times of the frontend quite a bit.
